### DISCUSSION TITLE
How to copy smth from disassembly view?

### DISCUSSION THREAD

**Discussion by FeelUsM** [microsoft/vscode-cpptools#12016] [Q&A] [answered]
When debugging I open disassembly view like this:
![image](https://private-user-images.githubusercontent.com/49320365/289141529-7096080a-77b6-4939-aff1-502f73a9803d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg4ODI0NDYsIm5iZiI6MTcwODg4MjE0NiwicGF0aCI6Ii80OTMyMDM2NS8yODkxNDE1MjktNzA5NjA4MGEtNzdiNi00OTM5LWFmZjEtNTAyZjczYTk4MDNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI1VDE3MjkwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmYjhjNmIwOThlODUxZGQ3YzFjOTE2YWMzMWMxNWE1MDFjZGU2YWQxYzU1OWQ2MTUxN2EwNjM0MjI1OWQ2OTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.TRUjn-AsS5hn0w5lHoDlZlRgxWkhuVLnTLI2i6lMTzE)
how to copy some text or some line from this page?

**Comment by bobbrow** [comment, marked as answer]
The disassembly view is part of the VS Code core editor.  We provide the data for that view when we have it, but we don't have access to the contents of the tab and can't control things like selection and copying.

**Reply by FeelUsM** [reply]
Why [disassembly view cannot be shown in the editor window](https://github.com/microsoft/vscode-cpptools/issues/11766)?
May be we should formulate a feature request directly to vscode?
